### ISSUE TITLE
do a v3.1.0 release?

### ISSUE THREAD

**Issue by UnixJunkie** [ocaml-batteries-team/batteries-included#963]

**Comment by facelesspanda** [comment]
It'd be nice to have #957 out there at least.

**Comment by UnixJunkie** [comment]
this has been merged so should be included

**Comment by UnixJunkie** [comment]
it is already included in batteries.3.0.0 by the way, which is already released.

**Comment by facelesspanda** [comment]
I do not find the commits in the history of the `v3.0.0` tag, so I assumed it wasn't. Are the github tags and the opam releases de-synchronized?

**Comment by UnixJunkie** [comment]
I found by looking at the changeLog, which says which PR was merged into each release.

**Comment by facelesspanda** [comment]
there was an error in the changelog amendments in #957 then, apparently. It adds (and I'm to blame, I think, for misunderstanding how this worked) the change to the list of changes in v3.0.0, which was already out (or frozen) by then, instead of under `NEXT_RELEASE`

See:
https://github.com/ocaml-batteries-team/batteries-included/pull/957/files

**Comment by UnixJunkie** [comment]
Ok, I'll correct it.

**Comment by UnixJunkie** [comment]
done
could be announced once this is merged:
https://github.com/ocaml/opam-repository/pull/17065
